newIDE/app/src/EffectsList/index.js

```diff
@@ -147,39 +147,49 @@ export default class EffectsList extends React.Component<Props, {||}> {
                   <React.Fragment key={i}>
                     <MiniToolbar>
                       <Column expand noMargin>
-                        <SemiControlledTextField
-                          margin="none"
-                          commitOnBlur
-                          hintText={t`Enter the effect name`}
-                          value={effect.getName()}
-                          onChange={newName => {
-                            if (newName === effect.getName()) return;
+                        <Line>
```

Comment:
This should probably be `<Line noMargin>` because vertical padding should be the responsibility of MiniToolbar already.

Also can you try to remove all Column/Line and see if it works?
```
<MiniToolbar>
  <MiniToolbarText>..</MiniToolbarText>
  < SemiControlledTextField >..</SemiControlledTextField >
  <MiniToolbarText>..</MiniToolbarText>
  < SemiControlledTextField >..</SemiControlledTextField >
```

Trying to see if we can simplify the markup

Reply:
Same thing with noMargin on <Line>.
Sightly better without <Column> and <Line> !

Reply:
Great! 👍